docs/Guide/application-commands/application-command-registry/registering-chat-input-commands.mdx

```diff
@@ -3,56 +3,174 @@ sidebar_position: 3
 title: Registering Chat Input Commands
 ---
 
-A chat input command can be registered to `Discord API` by letting the command registry know about its existence. To do
-so we need to `acquire` or use the `registerApplicationCommands` to have to `ApplicationCommandRegistry` to perform
-operations on it. Let's discuss about `ApplicationCommandRegistry` in more detail.
+A chat input command can be registered to the Discord API by letting the command registry know about its existence. To
+do so we need to acquire or use the [`registerApplicationCommands`][rac] to have access to the
+[`ApplicationCommandRegistry`][acr] to perform operations on it. You can learn how to do this on the page [Acquiring an
+Application Command Registry][aaacr].
+
+## How to register a chat input command
+
+The easiest way to register a Chat Input Command is to implement the [`registerApplicationCommands`][rac] method on a
```

Comment:
```suggestion
The easiest way to register a chat input command is to implement the [`registerApplicationCommands`][rac] method on a
```

Reply:
We capatilize it everywhere and I think that's better